frontend/src/Editor/QueryManager/CustomToggleSwitch.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
Can we move this into `components` folder?

Reply:
As discussed, for now keeping it inside query Manager component only